Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN apt-get update && \
 # Specify the version of Chrome that matches the version of chromedriver in the package.json.
 # A list of Chrome versions can be found here:
 # https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable
-ARG CHROME_VERSION=107.0.5304.121-1
+ARG CHROME_VERSION=107.0.3
```

Comment:
I don't believe this should be necessary - can you explain the purpose of rolling back the chrome version to an older one?

Reply:
Yes please, I will  delete this